maestro-studio/web/src/App.tsx

```diff
@@ -1,40 +1,24 @@
 import React, { useEffect, useState } from 'react';
 import './App.css';
-
-type TreeNode = {
-  attributes: {[key: string]: string}
-  children: TreeNode[]
-  clickable: boolean | undefined
-  enabled: boolean | undefined
-  focused: boolean | undefined
-  checked: boolean | undefined
-  selected: boolean | undefined
-}
-
-type Hierarchy = {
-  screenshot: string
-  tree: TreeNode
-}
+import { DeviceScreen } from './models';
+import Inspect from './Inspect';
 
 function App() {
-  const [hierarchy, setHierarchy] = useState<Hierarchy>()
+  const [deviceScreen, setDeviceScreen] = useState<DeviceScreen>()
   useEffect(() => {
     (async () => {
-      const response = await fetch('/api/hierarchy')
-      const hierarchy: Hierarchy = await response.json()
-      setHierarchy(hierarchy)
+      const response = await fetch('/api/device-screen')
+      const hierarchy: DeviceScreen = await response.json()
+      setDeviceScreen(hierarchy)
```

Comment:
Super Nit as we're moving away from hierarchy to device screen terminology
```suggestion
      const deviceScreenData: DeviceScreen = await response.json()
      setDeviceScreen(deviceScreenData)
```